assembl/static2/js/app/constants.js

```diff
@@ -31,4 +31,10 @@ ModeratedPublicationStates[PublicationStates.MODERATED_TEXT_ON_DEMAND] = Publica
 
 export const DeletedPublicationStates = {};
 DeletedPublicationStates[PublicationStates.DELETED_BY_USER] = PublicationStates.DELETED_BY_USER;
-DeletedPublicationStates[PublicationStates.DELETED_BY_ADMIN] = PublicationStates.DELETED_BY_ADMIN;
\ No newline at end of file
+DeletedPublicationStates[PublicationStates.DELETED_BY_ADMIN] = PublicationStates.DELETED_BY_ADMIN;
+
+export const CountablePublicationStates = {};
```

Comment:
Great! All good. One last thing I would add, since this code is duplicated across the front/back boundry, a comment to direct them to which file this is taken from, and a similar comment in Python, so any future changers would be clear that the refactoring has to be propagated across both files/languages. Otherwise, all good.